scrapers/prowlarr.py

```
@@ -562,22 +568,26 @@ async def process_stream(
 
         try:
             torrent_data, is_torrent_downloaded = await self.get_torrent_data(
                 download_url, prowlarr_data.get("indexer")
             )
-        except httpx.TimeoutException:
-            self.logger.warning("Timeout while getting torrent data")
+        except httpx.HTTPStatusError as error:
+            if error.response.status_code in [429, 500]:
+                raise error
+            self.logger.error(
+                f"HTTP Error getting torrent data: {error.response.text}, status code: {e.response.status_code}"
```

Ruff

Undefined name `e`

(F821)